scanpy/__init__.py

```diff
@@ -22,6 +22,7 @@
 
 # has to be done at the end, after everything has been imported
 import sys
+sys.modules.update({f'{__name__}.{m}': globals()[m] for m in ['tl', 'pp', 'pl']})
```

Comment:
Why was this added?

Reply:
This makes it so that `sc.tl` and so on is a real module python knows about, instead of a global variable that happens to point to a module object. Sphinx got more strict about where it allows imports from. It won’t use attribute access for something we claim is a module.

Also people can now do `from scanpy.tl import ...` which is consistent to what we claim in our modules (I.e. that `sc.tl` is a module)

Reply:
Is this better than having a `tl.py` which has: `from .tools import *`?

Reply:
yes, that way the modules are actually identical and not two different modules containing references to the same objects.